packages/react-dom/src/__tests__/ReactDOMNativeEventHeuristic-test.js

```diff
@@ -342,12 +343,11 @@ describe('ReactDOMNativeEventHeuristic-test', () => {
     });
     expect(container.textContent).toEqual('Count: 0');
 
-    const pressEvent = document.createEvent('Event');
-    pressEvent.initEvent('click', true, true);
-    dispatchAndSetCurrentEvent(target.current, pressEvent);
-    // Intentionally not using `act` so we can observe in between the press
```

Comment:
Aren't we breaking what this test is testing now? Before, we were intentionally not using act, and now we are?